content/docs/pipelines/overview/concepts/run.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Run and Recurring Run"
+title = "Run and Recurring Run (also known as a Job) "
```

Comment:
It would have been nice if KFP used the same term in the API and UI. I would recommend focusing on the UI terms for the title:

Run and Recurring Run 

Do you think that there are a significant set of users who primarily work with the API instead of the UI? If so, perhaps we should consider something like the following:

Run, and Recurring Run or Job

Reply:
That is a good point, I have only heard of one instance were this created confusion as the api name is hidden deep in the code and it is generally invisible to the users. However that being said there is no documentation that clarifies recurring run  is actually the same thing as the job API.  I'm ok with keeping the Run and Recurring Run title.